pkg/internal/checkup/trex/configfiles.go

```diff
@@ -0,0 +1,141 @@
+package trex
+
+import (
+	"fmt"
+
+	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/config"
+)
+
+const (
+	CfgFileName                = "trex_cfg.yaml"
+	StreamPyFileName           = "testpmd.py"
+	StreamPeerParamsPyFileName = "testpmd_addr.py"
+)
+
+type Configs struct {
+	masterCPU                      string
+	latencyCPU                     string
+	trafficCPUs                    string
+	numOfTrafficCPUs               string
+	portBandwidthGB                string
+	trafficGeneratorEastMacAddress string
+	trafficGeneratorWestMacAddress string
+	DPDKEastMacAddress             string
+	DPDKWestMacAddress             string
+}
+
+func NewTrexConfigs(cfg config.Config) Configs {
+	const (
+		masterCPU        = "0"
+		latencyCPU       = "1"
+		trafficCPUs      = "2,3,4,5,6,7"
+		numOfTrafficCPUs = "6"
+	)
+	return Configs{
+		masterCPU:                      masterCPU,
+		latencyCPU:                     latencyCPU,
+		trafficCPUs:                    trafficCPUs,
+		numOfTrafficCPUs:               numOfTrafficCPUs,
+		portBandwidthGB:                fmt.Sprintf("%d", cfg.PortBandwidthGB),
+		trafficGeneratorEastMacAddress: cfg.TrafficGeneratorEastMacAddress.String(),
+		trafficGeneratorWestMacAddress: cfg.TrafficGeneratorWestMacAddress.String(),
+		DPDKEastMacAddress:             cfg.DPDKEastMacAddress.String(),
+		DPDKWestMacAddress:             cfg.DPDKWestMacAddress.String(),
+	}
+}
+
+func (t Configs) GetTrexCfgFile() string {
```

Comment:
nit:
Please consider other verbs like `Generate` or `Create`.

Same for the other functions.

Reply:
DONE